src/LoginForm.jsx

```diff
@@ -0,0 +1,33 @@
+export default function LoginForm({ onChange, onSubmit, email, password }) {
+  return (
+    <div>
+      <form>
+        <div>
+          <label htmlFor="login-email">E-mail</label>
+          <input
+            id="login-email"
+            type="email"
+            name="email"
+            value={email}
+            placeholder="tester@example.com"
+            onChange={onChange}
+          />
+        </div>
+        <div>
+          <label htmlFor="login-password">Password</label>
+          <input
+            id="login-password"
+            type="password"
+            name="password"
+            placeholder="test"
+            value={password}
+            onChange={onChange}
+          />
+        </div>
+        <button type="button" onClick={onSubmit}>
+          Log In
+        </button>
+      </form>
```

Comment:
submit은 form에 적용하는게 좋아보입니다. e.preventDefault를 해야되긴 하지만, 엔터를 눌러도 제출이 되는 기능이 되기도 합니다
```suggestion
      <form onSubmit={onSubmit}>
        <div>
          <label htmlFor="login-email">E-mail</label>
          <input
            id="login-email"
            type="email"
            name="email"
            value={email}
            placeholder="tester@example.com"
            onChange={onChange}
          />
        </div>
        <div>
          <label htmlFor="login-password">Password</label>
          <input
            id="login-password"
            type="password"
            name="password"
            placeholder="test"
            value={password}
            onChange={onChange}
          />
        </div>
        <button type="submit">
          Log In
        </button>
      </form>
```

Reply:
엇 이건 e2e 인식에서 오류가나길래 타입이 submit이라서 안되는건가 하고 바꿔봤는데
알고보니 텍스트가 달라서 안되었던 부분이였네요ㅎㅎ..
수정했습니다!